src/core/configuration.c

```diff
@@ -408,34 +408,24 @@ QuicConfigurationParamGet(
         void* Buffer
     )
 {
-    QUIC_STATUS Status;
-
-    switch (Param) {
-    case QUIC_PARAM_CONFIGURATION_SETTINGS:
+    if (Param == QUIC_PARAM_CONFIGURATION_SETTINGS) {
```

Comment:
Isn't this a place where we're likely to add more options to the switch statement in the future? 

Also, I know multi returns is fine in C, but its definitely changing from what is mostly our standard. Definitely just want to double check that we're ok with this.

Reply:
I am fine with these changes. If we do need to add more in the future, it's easy enough to change then.